test/Common/DurableTaskEndToEndTests.cs

```diff
@@ -2951,7 +3038,7 @@ public async Task DurableEntity_ScheduledSignal(bool extendedSessions)
 
                 var now = DateTime.UtcNow;
 
-                await client.SignalEntity(this.output, now + TimeSpan.FromSeconds(1), "delayed", null);
+                await client.SignalEntity(this.output, now + TimeSpan.FromSeconds(2), "delayed", null);

```

Comment:
I'll open a PR to make this 5 seconds again, as 1 is a bit too aggressive I have found.

Reply:
Thanks! Yeah this was rather flaky. 1 is to few, 10 is too many.
I did have a flaky test with 5 seconds, so I suspect that somewhere around 2 to 4 might be a sweet spot. Aka, I'd try 3 :)

Reply:
It's flaky at 5 due to the fact the whole test stops at 10 seconds, and we already wait 5 seconds before we call signal.

Increasing that timeout accordingly should resolve flakiness.